spec/models/term_spec.rb

```diff
@@ -3,9 +3,22 @@
 require "rails_helper"
 
 RSpec.describe Term, type: :model do
+  #Test_Word :
```

Comment:
This comment is not necessary.
It does not improve the readability but also make code less attractive.

This comment style only works when:
- The code below the comment is hard to understand
```ruby
# Find a user with specific flag and transform it
User.all.select { |u| u.flag = :this_is_a_flag }.map { |u| u.name.upcase! }
```
- The data flow is complex
```ruby
# First, split the name by delimiter
klass_names = self.class.name.split("::")
# Second, transform them
klass_names.each { |string| string.downcase! }
# Finally, make the underscore
klass_names.join("_")
```

So, why is this comment bad?

```ruby
RSpec.describe Term, type: :model do # <- reader already understand this is a test
```
```ruby
it { should have_db_index :word } # <- this test is well-understand that for `word`
```